csvv.py

```diff
@@ -3,70 +3,71 @@
 import xarray as xr
 import pandas as pd
 import numpy as np
-from toolz import memoize
 from scipy.stats import multivariate_normal as mn
 
 
-class Gammas():
+def read_csvv(csvv_path):
+    '''
+    Returns the gammas and covariance matrix 
+    
+    Parameters
+    ----------
+    path: str
+        path to csvv file
+
+    Returns
+    -------
+    dict with keys of gamma, gammavcv, prednames, covarnames outcomes, and residvcv
+
+  	Extracts necessary information to specify an impact function
+    '''
+
+    data = {}
+
+    with open(csvv_path, 'r') as file:
+        reader = csv.reader(file)
+        for row in reader:
+            if row[0] == 'gamma':
+                data['gamma'] = np.array([float(i) for i in reader.next()])
+            if row[0] == 'gammavcv':
+                data['gammavcv'] = np.array([float(i) for i in reader.next()])
+            if row[0] == 'residvcv':
+                data['residvcv'] = np.array([float(i) for i in reader.next()])
+            if row[0] == 'prednames':
+                data['prednames'] = [i.strip() for i in reader.next()]
+            if row[0] == 'covarnames':
+                data['covarnames'] = [i.strip() for i in reader.next()]
+            if row[0] == 'outcome': 
+            	data['outcome'] =[cv.strip() for cv in reader.next()]
```

Comment:
if we can get the argument list to be consistent, then we could pass this into Gamma explicitly. It would be great if we could have a clearer signature, like

    class Gamma(object):
        def __init__(self, median, vcv, index):
            ...

and these attributes were used directly when sampling, e.g.:

    def median(self):
        return pd.Series(self.median, index=self.index).to_xarray()